app/components/leaflet.directive.js

```diff
@@ -136,6 +136,12 @@
         mapLayers['mobileMeasurements'].clearLayers();
         if (!angular.isDefined(newVal) || angular.equals({}, newVal)) return;
 
+        // find min & max values for color grading
+        var values = newVal.map(function(m) { return m.value; });
+        var max = Math.max.apply(null, values);
+        var min = Math.min.apply(null, values);
+        var palette = chroma.scale(['#B5F584', '#375F73']).mode('hcl').correctLightness().domain([max, min]);
```

Comment:
Let's use the d3 functionality for this and remove chroma-js

this could look like the following code:
```js
var color = d3.scaleLinear()
      .domain([1,100]) // min and max
      .interpolate(d3.interpolateHcl)
      .range([d3.rgb("#B5F584"), d3.rgb('#375F73')]);
```

For details about the used d3 functionalities:
https://github.com/d3/d3-color/blob/master/README.md#color
https://github.com/d3/d3/blob/master/API.md#scales-d3-scale
https://github.com/d3/d3-interpolate

Reply:
The point of using `chroma` is that it produces equiluminant colors (`.correctLightness()`) - which I concluded in my thesis to be far better suited for dataviz purposes, because they more comparable.
See https://earthobservatory.nasa.gov/blogs/elegantfigures/2013/08/06/subtleties-of-color-part-2-of-6/

If you think this is not worth the additional  33kb, we can use plain d3 colors ofc

Reply:
`d3.interpolateHcl()` returns nice enough results 👍